src/sqlite.rs

```diff
@@ -249,15 +253,17 @@ pub fn open(package_id: PackageId, db: &str) -> anyhow::Result<Sqlite> {
 }
 
 /// Remove and delete sqlite database.
-pub fn remove_db(package_id: PackageId, db: &str) -> anyhow::Result<()> {
+pub fn remove_db(package_id: PackageId, db: &str, timeout: Option<u64>) -> anyhow::Result<()> {
```

Comment:
we have some more of these functions in src/vfs/mod.rs, like fn metadata() , and remove_path(), should they take optional timeouts as well? 

I think they should stay outside of structs for simple use

Reply:
Good question.. let's sleep on it and keep this open for now.